### PR TITLE
ceph-dev-new-setup: Run on any node

### PR DESCRIPTION
**Pull request by zmc**

### CODE REVIEW

File: ceph-dev-new-setup/build/build [hunk 1]
```diff
@@ -103,10 +103,18 @@ echo current version $cephver
...
+else
+    PODMAN="sudo docker"
+fi
+
 if [ -d "$releasedir/$cephver" ]; then
     echo "$releasedir/$cephver already exists; reuse that release tarball"
 else
-    dch -v $cephver-1 'autobuilder'
+    # Create a container image to provide debian-specific utilities, so that this job can run on any container-capable host
+    printf "FROM ubuntu:24.04\nRUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y dpkg-dev devscripts && apt-get clean && rm -rf /var/lib/a^C/lists/*" | $PODMAN build -t ubuntu_builder -
```
Comment (dmick) [inline ceph-dev-new-setup/build/build:116]: must be a typo in the rm -rf (^C)?

**Review by dmick — COMMENTED** [review]
How have you tested (apologies if you already told me)

File: ceph-dev-new-setup/build/build (hunk 1 above)
Comment (zmc) [reply ceph-dev-new-setup/build/build]: whoa, good catch. Repushed.

**Comment by zmc** [comment]
I tested this by pushing it to jenkins while builds were idle and performing some builds. I saw it succeed on centos and ubuntu hosts.

**Review by dmick — APPROVED** [review]

**Pull request merged**